### ISSUE TITLE
Intermodal Bike+ÖPNV routes with RegioRad should end at a RegioRad station (web and app)

### ISSUE THREAD

**Issue by stadtnavimanager** [stadtnavi/digitransit-ui#861]
**Description:**  

When **RegioRad** is enabled in the settings, intermodal **Bike+Public Transport (Bike+ÖPNV)** routes include suggestions that combine **Public Transport (ÖPNV)** and **bike-sharing (RegioRad)**.  

However, these routes incorrectly end at the **entered destination**, rather than at a designated **RegioRad station** where the bike must be returned. This can mislead users and create practical issues for completing the trip.

---

**Steps to reproduce (web):**  

1. Open stadtnavi and search route, e.g.: https://dev.stadtnavi.eu/reiseplan/Nauener%20Stra%C3%9Fe%2029%2C%2070597%20Stuttgart%3A%3A48.7329079%2C9.178253117453703/Rosenbrunnenstra%C3%9Fe%2C%2070771%20Unteraichen%3A%3A48.7052866%2C9.1399363/bikeAndVehicle?setTime=true&time=1734540038
2. enable **RegioRad** in the **Settings** menu.  
3. Search for an intermodal **Bike+ÖPNV** route.  
4. Observe the suggested routes that include RegioRad.  

---

**Observed behavior:**  

The suggested route with RegioRad ends at the user’s destination instead of a valid RegioRad return station.  

**Expected behavior:**  

Routes that include RegioRad should end at a **RegioRad station** near the destination, where the shared bike can be returned.

---

**Screenshots:**  

![Image](https://github.com/user-attachments/assets/31c288fb-edd0-40db-862c-b6e0383e9302)

---

**Additional information:**  

Correcting this issue will ensure that users are provided with realistic and actionable route suggestions, improving the reliability of intermodal routes involving bike-sharing services.

**Comment by hbruch** [comment]
This is a feature, not a bug. We allow taking the bike to the destination and give a hint, that rental can not be terminated at the destination. Otherwise, in Herrenberg, we would never suggest RegioRad rental, as there is only on rental station at the station.

**Comment by MartinH-open** [comment]
> This is a feature, not a bug. 

Yes, I actually asked a RegioRad manager about such use case with only one RegioRad station in range.
If you rent the bike from there and keep it over night to return it back to the station next morning (before some early time like 9 am?) then you just pay a small additional fee to keep the bike occupied over night. Quite nice if you return home late and use the rental bike for the last mile home and you can return it back next morning. (Such use case should be mentioned - at least if you rent a bike in the evening time.)

**Comment by stadtnavimanager** [comment]
Thanks for the feedback. The station in Herrenberg unfortunately will be closed very soon.

I think even with the (valid) argument, the route suggestions right now are not ideal.
1. in case that there is a station close to the destination, stadtnavi could suggest to leave the bike there
2. i testet a route which is basically from station to station - here stadtnavi does not suggest renting a RegioRad at all:

https://dev.stadtnavi.eu/reiseplan/Gro%C3%9Fe%20Falterstra%C3%9Fe%2C%2070597%20Stuttgart%3A%3A48.74157464797559%2C9.17578347730451/Karl-Klo%C3%9F-Stra%C3%9Fe%2C%2070199%20Stuttgart%3A%3A48.76171758900294%2C9.159891100984813?time=1734598917

![image](https://github.com/user-attachments/assets/3c6dbf77-c44d-44bc-a60d-35be785e374e)

We should discuss this, also together with this issue: https://github.com/orgs/stadtnavi/projects/8?pane=issue&itemId=91389658&issue=stadtnavi%7Cdigitransit-ui%7C860
